### ISSUE TITLE
kiwi module kiwi_cmd 'kerberos::list /export'

### ISSUE THREAD

**Issue by s-e-knudsen** [rapid7/metasploit-framework#15153]
## Summary
Exporting the kerberos ticket list using the kiwi_cmd 'kerberos::list /export' do not save the kirbi file.  

## Relevant information
When running the module load kiwi and the the command kiwi_cmd 'kerberos::list /export' to export the list it states that the data is saved to file: <filename>, but the file do not exists in the file system. 

The rb code states that the file location "default" should be . but no file in the current directory.

I also tried to use the command to specify a directory using the command

kiwi_cmd 'kerberos::list /export -p /home/kali' but here the same not file saved. 

I might be doing somthing wrong or ?

**Comment by smcintyre-r7** [comment]
[Looks](https://github.com/rapid7/mimikatz/blob/9339edbaa10a2762ca149adffbb9c5372882860c/mimikatz/modules/kerberos/kuhl_m_kerberos.c#L242-L263) to me like the file is written to disk on the **remote** system, that is the Windows system on which Meterpreter is running. The file would be written into the current working directory of Meterpreter and then you'd need to download.

Have you checked the remote directory?

**Comment by OJ** [comment]
This is one of those cases where we probably need to expose the function as an actual feature in Meterp, because we need to have the output returned via base64 so that we can avoid writing stuff to disk.

**Comment by s-e-knudsen** [comment]
Sorry about the delay. 

Here is som information on the setup I did for this test. 
Created a simple metterperter windows/x64 called met11.exe

Win10 
Run the met11.exe in a cmd

The from the Meterpreter used the load kiwi and described above. 

After this I did a look at at kali and win10 for the file kirbi. 
I did look at the current directory on the win10 (where I started the met11.exe) and did a search for the file name on all diske both win10 and kali. File not available. 

I do agree with OJ. This would be good to save on the kali so the target disk is not used. Better for AV evasions. 

/Soren
